pages.de/common/date.md

```diff
@@ -0,0 +1,36 @@
+# date
+
+> Setze die Systemzeit oder zeige sie an.
+> Weitere Informationen: <https://www.gnu.org/software/coreutils/date>.
+
+- Zeige das aktuelle Datum im eingestellten Locale Format an:
+
+`date +%c`
+
+- Zeige das aktuelle Datum in koordinierter Weltzeit (UTC) im ISO 8610-Format an:
+
+`date -u +%Y-%m-%dT%H:%M:%S%Z`
+
+- Zeige das aktuelle Datum in Unixzeit (vergangene Sekunden seit der Unix-Epoche) an:
+
+`date +%s`
+
+- Konvertiere ein in Unixzeit gegebenes Datum zum Standardformat:
+
+`date -d @{{1473305798}}`
+
+- Konvertiere ein im Standardformat gegebenes Datum zu Unixzeit:
```

Comment:
The example isn't standard format, that would be something like `Sat Sep  1 00:00:00 CEST 2018`.

```suggestion
- Konvertiere ein gegebenes Datum zu Unixzeit:
```